string time
string c_cmd
